README.md

```diff
@@ -1,2 +1,7 @@
-# Rent a House
-This app will allow you to rent/buy a house.
+# (about/README) Rent a House
+Key: BREAD= browse, read, edit, add, delete. This website will allow you to rent/buy a house.
```

Comment:
This sounds like the BREAD information has to be in a different section. On top of that is not something that the final user will care about, and if it is a message to a developer, he/she/it should know this information, maybe it will be more useful to specify it on an _Architectural specification document. But not in here :)